main.go

```diff
@@ -86,7 +89,7 @@ func entrypoint(shutdown chan os.Signal) {
 		"oasApiPath":  env.TargetServiceOASPath,
 	}).Trace("OAS successfully loaded")
 
-	mongoClient, err := mongoclient.NewMongoClient(env, log)
+	mongoClient, err := mongoclient.NewMongoClient(env, rondlogrus.NewLogger(log))
```

Comment:
Why aren't you submitting the same logger created at line [73](https://github.com/rond-authz/rond/pull/220/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R73)?

Reply:
Used the same